presto-main/src/main/java/com/facebook/presto/operator/HashAggregationOperator.java

```diff
@@ -309,7 +309,7 @@ private void processPage(Page page)
             }
         }
 
-        public boolean isFull()
+        public boolean checkFullAndUpdateMemory()
```

Comment:
I agree with you that this method should be renamed. I'll think about it and see if there's a better name.